components/vector-kubearchive-log-collector/staging/stone-stg-rh01/vector-helm-values.yaml

```diff
@@ -57,51 +58,27 @@ customConfig:
         if exists(.tmp.kubernetes.pod_namespace) {
           .namespace = del(.tmp.kubernetes.pod_namespace)
         } else {
-          .namespace = "unlabeled"
+          .namespace = "unknown_namespace"
         }
-        # Handling Tekton-specific labels
-        if exists(.tmp.kubernetes.pod_labels."tekton.dev/taskRunUID") {
-          .taskRunUID = del(.tmp.kubernetes.pod_labels."tekton.dev/taskRunUID")
+        # Preserve the actual log message - this was missing!
+        if exists(.tmp.message) {
+          .message = to_string(del(.tmp.message)) ?? "no_message"
         } else {
-          .taskRunUID = "none"
+          .message = "no_message"
         }
-        if exists(.tmp.kubernetes.pod_labels."tekton.dev/pipelineRunUID") {
-          .pipelineRunUID = del(.tmp.kubernetes.pod_labels."tekton.dev/pipelineRunUID")
-          .result = .pipelineRunUID
-        } else {
-          .result = .taskRunUID
-        }
-        # --- Start: Cronjob Specific Handling ---
-        if exists(.tmp.kubernetes.pod_labels."job-name") {
-          .job_name = del(.tmp.kubernetes.pod_labels."job-name")
-          .log_type = "cronjob"
-          if exists(.tmp.kubernetes.pod_labels."cronjob-name") {
-            .cronjob_name = del(.tmp.kubernetes.pod_labels."cronjob-name")
-          } else {
-            # Using corrected regex pattern without \d
-            .job_name = to_string(.job_name) ?? "default"
-            if match(.job_name, r'^(.*)-[0-9]{8,10}$') {
-              .cronjob_name = replace(.job_name, r'-[0-9]{8,10}$', "")
-            } else {
-              .cronjob_name = "unknown_cronjob"
-            }
-          }
-          if exists(.tmp.kubernetes.pod_labels."controller-uid") {
-              .job_uid = del(.tmp.kubernetes.pod_labels."controller-uid")
-          }
-        } else {
-          .log_type = "application"
+        # Basic data sanitization to prevent 400 errors
+        # Truncate very long messages
+        if length(.message) > 32768 {
+          .message = slice!(.message, 0, 32768) + "...[TRUNCATED]"
```

Comment:
We should keep an eye on this, we want to have the whole logs, not a truncated version of them but I think it's better to have the truncated version than 400 messages with no logs at all.

Let's keep an eye on how things go in staging and if we need to tweak this number or the maximum chunk size in loki.

Reply:
yes and it's ok, it's better to have it there than to miss logs, but if people start getting truncated logs from the konflux UI we need to revisit this